activities/Moon.activity/css/activity.css

```diff
@@ -145,7 +145,7 @@ canvas {
 }
 
 .tutorial-end-icon2 {
-	background-image: url(../icons/dialog-cancel.svg);
+	background-image: url(../../../icons/dialog-cancel.svg);
```

Comment:
Not at all a good idea to rely on icons in Sugarizer Core. Each activity should be deployable by itself